go/libkb/chain_link_v2.go

```diff
@@ -62,6 +62,8 @@ const (
 // be stubbed, that means we potentially won't get to verify its signature,
 // since we need the full link to verify signatures. However, in some cases,
 // signature verification is required, and hence stubbing is disallowed.
+// NOTE when modifying this function ensure that web/sig.iced#_allow_stubbing
+// is updated as well.
```

Comment:
Do we want to add any other types to allow stubbing? https://github.com/keybase/client/pull/13282/files#diff-b4ef7cfc80144dbd68006e618f836ccaR78

Reply:
i think this list is good for now.